src/components/Notifications/utils.ts

```diff
@@ -61,7 +61,7 @@ export const getGqlPayload = (
     trigger,
     extraFields,
     regexSelectors,
-  } = event;
+  } = event || {};
```

Comment:
Had to make these changes to handle the above case with the default values being undefined.